packages/component/src/BasicTranscript.js

```diff
@@ -44,6 +45,8 @@ const {
   useStyleOptions
 } = hooks;
 
+const SCROLL_RELATIVE_DELTA_IN_PIXEL = 40;
```

Comment:
Hm I'm confused about this. Don't we want to scroll by element because of AT?

Reply:
The mouse wheel test I tried in Chromium Edge is 40px, that's why I stick to this number.